airiam/find_unused/RuntimeIamScanner.py

```diff
@@ -12,6 +14,16 @@
 ERASE_LINE = '\x1b[2K'
 
 
+def get_iam_data_file(account_id=None):
+    if account_id is None:
```

Comment:
@manishsingh10895 When would the account_id be `None`? It seems you always pass in the account_id from the top - and it makes sense, as the worst is wrong data. Why keep this backwards compatibility?

Reply:
I might be wrong in this but, I believe `self._get_identity_details()` method can return empty strings when **profile** passed in arguments is invalid. 

Python is not my first language, so again I might be wrong in this.

Reply:
@manishsingh10895 if the profile is passed in incorrectly, this will fail:
```python
account_id, identity_arn = self._get_identity_details()
```
And your lines will not be reached. You should be good IMO

Reply:
So, can you help me with what should I do here?. 
Should I remove the None default and None check in the function and open another pull request after closing this one.

Reply:
Sure. Writing a suggestion